include/cpp-statsd-client/UDPSender.hpp

```diff
@@ -213,7 +225,7 @@ inline bool UDPSender::initialize() noexcept {
             // An error code has been returned by getaddrinfo
             close(m_socket);
             m_socket = k_invalidFd;
-            m_errorMessage = "getaddrinfo failed: error=" + std::to_string(ret) + ", msg=" + gai_strerror(ret);
+            m_errorMessage = "getaddrinfo failed: err=" + std::to_string(ret) + ", msg=" + gai_strerror(ret);
```

Comment:
i tried to make all the error message look as much like each other as possible. its still not perfect but slightly better i guess